### DISCUSSION TITLE
【委托】星际和平公司重金悬赏████「██」

### DISCUSSION THREAD

**Discussion by siler-wolf** [share121/inter-knot#534] [其他]
![Uploading c19aa44b1e976ba6e5aeeb1e63719c83.png…]()
此人乃「████」的一员，曾参与过████犯下的重多案件，且是一名骇客，十分危险。如遇见或抓获，请立刻通知星际和平公司，可获得51亿的奖金。
「公司员工把通缉令都发布到这来了，真是厉害。我倒是好奇,抓不到我，他们发这个又有什么用?」
「不管了，反正只要把通缉令处理一下就好了」

**Comment by xinxingjie** [comment]
我一眼就看出是谁的手笔，就是你吧，bro妮娅！

**Comment by 1155993355** [comment]
这个委托我接了

**Comment by siler-wolf** [comment]
改了 但在绳网显示不了

**Comment by Tragedysaver1024** [comment]
51亿奖金？单位是丁尼还是菲林？

**Reply by Nirvana341** [reply]
> 51亿奖金？单位是丁尼还是菲林？

要是给你整成信用点，这边还用不了。这种跨世界的委托还是看看就好，不值

**Reply by siler-wolf** [reply]
> 51亿奖金？单位是丁尼还是菲林？

丁尼

**Reply by FrostPhoenix79** [reply]
万一是信用点就赚了啊
